staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/condition_test.go

```diff
@@ -999,17 +1000,16 @@ func TestRuntimeCELCostBudget(t *testing.T) {
 	}
 
 	cases := []struct {
-		name                        string
-		attributes                  admission.Attributes
-		params                      runtime.Object
-		validations                 []ExpressionAccessor
-		hasParamKind                bool
-		authorizer                  authorizer.Authorizer
-		testRuntimeCELCostBudget    int64
-		exceedBudget                bool
-		expectRemainingBudget       *int64
-		enableStrictCostEnforcement bool
```

Comment:
updated the testcases that left this non-strict to set budgets that match their strict costs, rather than deleting the non-strict testcases altogether